dashboard/lib/service/appengine_cocoon.dart

```diff
@@ -249,6 +307,21 @@ class AppEngineCocoonService implements CocoonService {
     return result;
   }
 
+  CommitDocument _commitFromJsonFirestore(Map<String, dynamic> jsonCommit) {
+    final CommitDocument result = CommitDocument()
+      ..documentName = jsonCommit['DocumentName']
+      ..createTimestamp = Int64() + jsonCommit['CreateTimestamp']!
```

Comment:
This line is odd, can you just instantiate the Int64 with the jsonCommit value here?

Reply:
Yeah! See your code below.

Reply:
Fixed!